test/statetest/statetest_loader.cpp

```diff
@@ -140,6 +140,20 @@ inline uint64_t calculate_current_base_fee_eip1559(
     return base_fee;
 }
 
+namespace
+{
+bytes32 load_bytes32(const json::json::const_iterator& json_it)
```

Comment:
```suggestion
bytes32 load_bytes32(const json::json& j)
```
Just take the "json" because it is unconditionally dereferenced anyway.